nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -181,19 +181,43 @@ Initial Storage Level|Electricity|Energy Storage System:
    unit: GWh
 
 Maximum Charge|Electricity|Energy Storage System:
-   description: Maximum charge when storing energy the energy storage system unit.
-   unit: MW
+   description: Maximum charge when storing energy in the energy storage system unit.
+   unit: GW
 
 Maximum Discharge|Electricity|Energy Storage System:
    description: Maximum discharge when consuming energy from the energy storage system unit.
-   unit: MW
+   unit: GW
+
+Maximum Charge|Electricity|Energy Storage System|Lithium-Ion:
```

Comment:
It would make sense to start a new type-list like we have for fuels... Also, question is whether pumped-storage should be part of this naming convention or have its own structure, see #102 - @erikfilias, what do you think?

Reply:
I also agree with a tag for all types of energy storage. They have in common the following characteristics:
- Maximum Charge
- Maximum Discharge
- Initial Storage
- Maximum Storage
- Minimum Storage
- Efficiency
- Storage type or cycle (daily, weekly, or monthly)
Also, we have the known common variables for all generating unit like:
- Maximum Power
- Minimum Power

So, it can help us to start a new tag. 
Even a hydro unit with a reservoir could be considered for this tag according to an inventory constraint (energy management into an operation cycle or storage type).

Reply:
Okay, @danielhuppmann do you think that this should rather be done in a new PR instead of modifying this PR?

Reply:
@erikfilias from my point of view the variables are different for i/thermal units (the ones for which we have already included the <fuel> tag, ii/energy storage system : yes would be interested to have a <techno> tag (with Lithium-Ion, Compressed Air.....  ii/ Pumped Storage : this is a storage system but it is slightly different, I would not integrate it into the list of energy storage systems at this stage 
Other comment: Maximum power and Minimum Power do not exist for energy storage system, where we have Maximum Charge/Discharge (even if in the end those are equal); efficiency is not there neither (used for hydro, but for energy storage systems there is RoundTrip coefficient)
And finally a Hydro Unit with a reservoir has additionnal characteristics such as inflows... 
We could think of merging all this but it would require discussing it (and would induce some delays in implementation of models as some case studies require that datasets are built quite soon, so we need the nomenclature to be quite stable now, ie what would be great is to limit to minor changes + add variables + correct bugs but not do many major restructurations :-)

Reply:
I also agree with @sandrinecharousset  that this implementation needs a discussion and induces some delays.
For this, we can continue with this PR and #102.
And to put this new tag as an issue for a while.